src/PanelGroup.js

```diff
@@ -96,11 +90,13 @@ class PanelGroup extends React.Component {
     };
   }
 
-  handleSelect(key, expanded, e) {
+  handleSelect = (key, expanded, e) => {
     if (expanded) {
       this.props.onSelect(key, e);
+    } else if (this.props.activeKey === key) {
```

Comment:
You should be able to click on a panel in an accordion again to close it.

I'm not sure setting `activeKey` to `null` is the best way to do this, though. It'd be weird to use `undefined`.